app/frontend/src/features/messages/utils.tsx

```diff
@@ -49,12 +51,17 @@ export function groupChatTitleText(
   groupChatMembersQuery: ReturnType<typeof useUsers>,
   currentUserId: number
 ) {
-  return groupChat.title
-    ? groupChat.title
-    : groupChatMembersQuery.isLoading
-    ? "Chat"
-    : Array.from(groupChatMembersQuery.data?.values() ?? [])
-        .filter((user) => user?.userId !== currentUserId)
-        .map((user) => firstName(user?.name))
-        .join(", ");
+
+
+  if (groupChat.title) {
+    return groupChat.title;
+  } else if (groupChatMembersQuery.isLoading) {
+    return "Chat";
+  }
+
+  return Array.from(groupChatMembersQuery.data?.values() ?? [])
+    .filter((user) => user?.userId !== currentUserId)
+    .map((user) => <Link to={routeToUser(user?.username || "")}>{firstName(user?.name)}</Link>)
+    .reduce((arr: any[] | null, element: any) => arr === null ? [element] : [...arr, ', ', element], null);
```

Comment:
I know this reduce in here is a little funky, found it here https://stackoverflow.com/questions/33577448/is-there-a-way-to-do-array-join-in-react/44738250 maybe someone knows of a better way to join the list of links?